src/main/java/bio/terra/flight/file/delete/FileDeleteFlight.java

```diff
@@ -28,15 +27,12 @@ public FileDeleteFlight(FlightMap inputParameters, Object applicationContext, Us
         DataLocationService locationService = (DataLocationService) appContext.getBean("dataLocationService");
 
         // TODO: fix this
-        // There are two problems here.
-        // First, the PATH_PARAMETERS debacle.
-        // Second, error handling within this constructor results in an obscure throw from
+        // Error handling within this constructor results in an obscure throw from
         // Java (INVOCATION_EXCEPTION), instead of getting a good DATASET_NOT_FOUND error.
         // We should NOT put code like that in the flight constructor.
```

Comment:
This comment refers to retrieving the dataset below, not to extracting the parameters.
I added DR-619 to track the INVOCATION_EXCEPTION issue.

Reply:
Same reply as above. Will fix and update PR.